packages/components/src/components/linear-progress/linear-progress.tsx

```diff
@@ -1,7 +1,13 @@
 import classNames from 'classnames';
 import React from 'react';
 
-export const LinearProgress = ({ progress }) => (
+type TLinearProgress = {
+    progress: number;
+    className: string;
+    height: number;
+};
```

Comment:
why do we need  className and  height in this component? I couldn't find using them in the component.